src/StoreKeeper/WooCommerce/B2C/Backoffice/Helpers/OverlayRenderer.php

```diff
@@ -79,6 +91,66 @@ public function renderError($message, $description = '')
         $this->renderContentItem($content);
     }
 
+    public function renderMemoryExhaustError($message, $description = '')
+    {
+        $message = esc_html($message);
+        $description = esc_html($description);
+
+        $firstExplanationText = __('Your allowed memory size has been exhausted. Here are some solution that may solve the issue.', I18N::DOMAIN);
+        $secondExplanationText = __('But before trying some of these you can check what is the defined memory_limit of your site. You can check this by Clicking Tools > Site Health > Info > Server. Here you can check the current settings of your hosting.', I18N::DOMAIN);
+
+        $serverInfoImagePath = plugin_dir_url(__FILE__).'../static/server-information.png';
```

Comment:
We can easily read out what is the current memory setting, instead of asking the user to check other page which will close the message :P

Reply:
Read from ini and removed image.